### PR TITLE
fix multiple bugs #1 #47 #204

### PR DESCRIPTION
**Pull request by parthdalaltech**
## Description

fixed :
https://github.com/sparrowapp-dev/sparrow-app/issues/47
https://github.com/sparrowapp-dev/sparrow-app/issues/1
https://github.com/sparrowapp-dev/sparrow-app/issues/207

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [X] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [X] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [X] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

**Comment by punitkashyup** [comment]
### SonarQube Quality Gate Result
- **Result**: :white_check_mark: OK
- Triggered by @parthdalaltech on `pull_request`

| Metric | Status | Value | Error Threshold |
|:------:|:------:|:-----:|:---------------:|
|New reliability rating|:white_check_mark: OK|1|> 1|
|New security rating|:white_check_mark: OK|1|> 1|
|New maintainability rating|:white_check_mark: OK|1|> 1|

[View on SonarQube](https://sonar.techdomeaks.com/dashboard?id=sparrow)
###### _updated: 2/26/2024, 06:14:20 (UTC+0)_

File: src/lib/components/dashboard/workspace-card-list/WorkspaceCardList.svelte [hunk 1]
```diff
@@ -97,8 +97,9 @@
           type="other"
           buttonClassProp={`rounded sparrow-fs-16 col-lg-5 col-md-10 flex-grow-1 py-0 mb-4 add-new-workspace`}
           onClick={handleCreateWorkspace}
-          buttonStyleProp={workspaces.length == 0 ? "min-height:18vh" : ""}
+          buttonStyleProp={workspaces.length == 0 ? "min-height:132px" : ""}
```
Comment (LordNayan) [inline src/lib/components/dashboard/workspace-card-list/WorkspaceCardList.svelte:100]: can we use calc here

File: src/lib/components/header/global-search-bar/GlobalSearchBar.svelte [hunk 2]
```diff
@@ -364,6 +364,7 @@
     font-weight: 500;
     margin-right: 8px;
     text-align: left;
+    width: 40px;
```
Comment (LordNayan) [inline src/lib/components/header/global-search-bar/GlobalSearchBar.svelte:367]: Can do it in percentage.

File: src/App.svelte [hunk 3]
```diff
@@ -24,6 +24,7 @@
   export let url = "/";
   const tabRepository = new TabRepository();
   let flag: boolean = true;
+  let isActiveInternet: boolean = true;
```
Comment (LordNayan) [inline src/App.svelte:27]: no need to specify type here.

let isActiveInternet = true; should do

File: src/lib/components/dashboard/workspace-card-list/WorkspaceCardList.svelte (hunk 1 above)
Comment (parthdalaltech) [reply src/lib/components/dashboard/workspace-card-list/WorkspaceCardList.svelte:100]: kept it for now and all comment to check later . in my todo list already

File: src/lib/components/header/global-search-bar/GlobalSearchBar.svelte (hunk 2 above)
Comment (parthdalaltech) [reply src/lib/components/header/global-search-bar/GlobalSearchBar.svelte:367]: this is fixed value for the icons. percentage is varying due to resize.

File: src/App.svelte (hunk 3 above)
Comment (parthdalaltech) [reply src/App.svelte:27]: followed the trend :-P
will update in next PR

Comment (LordNayan) [reply src/App.svelte:27]: okk

**Review by LordNayan — APPROVED** [review]

**Pull request merged**